packages/synapse-react-client/src/components/GoalsV2/GoalsV2.tsx

```diff
@@ -0,0 +1,235 @@
+        )
+        if (assetFileHandleIds.length === 0) {
+          // wait for data to load
+          return
+        }
+        const fileHandleAssociationList: FileHandleAssociation[] =
+          assetFileHandleIds.map(fileId => {
+            return {
+              associateObjectId: entityId,
+              associateObjectType: FileHandleAssociateType.TableEntity,
+              fileHandleId: fileId,
+            }
+          })
+        const batchFileRequest: BatchFileRequest = {
+          includeFileHandles: false,
+          includePreSignedURLs: true,
+          includePreviewPreSignedURLs: false,
+          requestedFiles: fileHandleAssociationList,
+        }
+        const files = await getFiles(batchFileRequest, accessToken)
+        setError(undefined)
+        setAssets(
+          files.requestedFiles
+            .filter(el => el.preSignedURL !== undefined)
+            .map(el => el.preSignedURL!),
+        )
+      } catch (e) {
+        console.error('Error on get data', e)
+        setError(e)
+      }
+    }
+    getData()
+  }, [entityId, accessToken, queryResultBundle])
+
+  const tableIdColumnIndex = getFieldIndex(
+    ExpectedColumns.TABLEID,
+    queryResultBundle,
+  )
+  const countSqlColumnIndex = getFieldIndex(
+    ExpectedColumns.COUNT_SQL,
+    queryResultBundle,
+  )
+
+  const titleColumnIndex = getFieldIndex(
+    ExpectedColumns.TITLE,
+    queryResultBundle,
+  )
+  const summaryColumnIndex = getFieldIndex(
+    ExpectedColumns.SUMMARY,
+    queryResultBundle,
+  )
+  const linkColumnIndex = getFieldIndex(ExpectedColumns.LINK, queryResultBundle)
+
+  const goalsDataArray: GoalsV2DataProps[] = []
+
+  queryResultBundle?.queryResult!.queryResults.rows.forEach((el, index) => {
+    const values = el.values as string[]
+    if (values.some(value => value === null)) {
+      console.warn('Row has null value(s) when no nulls expected')
+    }
+    const tableId =
+      tableIdColumnIndex > -1 ? values[tableIdColumnIndex] : undefined
+    let countSql
+    if (countSqlColumnIndex > -1 && values[countSqlColumnIndex]) {
+      countSql = values[countSqlColumnIndex]
+    } else if (tableId) {
+      countSql = `SELECT * FROM ${tableId}`
+    }
+    const title = values[titleColumnIndex]
+    const summary = values[summaryColumnIndex]
+    const link = values[linkColumnIndex]
+    const asset = assets?.[index] ?? ''
+    const goalsV2DataProps: GoalsV2DataProps = {
+      countSql,
+      title,
+      summary,
+      link,
+      asset,
+    }
+    goalsDataArray.push(goalsV2DataProps)
+  })
+
+  return (
+    <Box
+      sx={{
+        backgroundColor: 'rgba(0, 128, 0, 0.05)',
+        height: '560px',
+        padding: '80px',
+      }}
+    >
+      <Box
+        sx={{
+          textAlign: 'center',
+          paddingBottom: '30px',
+        }}
+      >
+        <Typography
+          variant="headline1"
+          style={{
+            color: 'grey.1000',
+            fontFamily: 'Merriweather',
```

Comment:
Should be unnecessary now that this is in the HomePageThemeProvider in the EL Portal
```suggestion
```